### PR TITLE
docs(common):update csp docs

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: troubleshooting/csp.md [hunk 1]
```diff
@@ -32,5 +35,12 @@ These limitations can adversely affect the Telerik UI for Blazor components, bec
...
 ````
 
->tip If you do not use our CDN services, you can remove their domains. If you do not use the templates of the charts, you may also be able to remove `'unsafe-eval'`.
+>tip If you do not use our CDN services, you can remove their domains. If you do not use the Templates of the Charts, you may also be able to remove `'unsafe-eval'`.
+
+## Upcoming Enhancements
+
+Some of the above-listed limitations will be addressed in a future version of Telerik UI for Blazor. The planned enhancements are:
 
+* Remove the necessity for specifying the `unsafe-eval` directive when the [Chart Templates]({%slug components/chart/label-template-format%}) are used.
```
Comment (dimodi) [inline troubleshooting/csp.md:44]:
```suggestion
* Remove the need to specify `unsafe-eval` when [Chart Templates]({%slug components/chart/label-template-format%}) are used.
```

File: troubleshooting/csp.md [hunk 2]
```diff
@@ -32,5 +35,12 @@ These limitations can adversely affect the Telerik UI for Blazor components, bec
...
+* Remove the necessity for specifying the `unsafe-inline` directive.
```
Comment (dimodi) [inline troubleshooting/csp.md:45]:
```suggestion
* Remove the need to specify `unsafe-inline`.
```

File: troubleshooting/csp.md [hunk 3]
```diff
@@ -32,5 +35,12 @@ These limitations can adversely affect the Telerik UI for Blazor components, bec
...
+* Fix the `font-src` rule by detaching font icons.
```
Comment (dimodi) [inline troubleshooting/csp.md:46]: What exactly do we fix?

**Review by dimodi — APPROVED** [review]

File: troubleshooting/csp.md [hunk 4]
```diff
@@ -32,5 +35,12 @@ These limitations can adversely affect the Telerik UI for Blazor components, bec
...
->tip If you do not use our CDN services, you can remove their domains. If you do not use the templates of the charts, you may also be able to remove `'unsafe-eval'`.
+>tip If you do not use our CDN services, you can remove their domains. If you do not use the Templates of the Charts, you may also be able to remove `'unsafe-eval'`.
+
+## Upcoming Enhancements
+
+Some of the above-listed limitations will be addressed in a future version of Telerik UI for Blazor. The planned enhancements are:
 
+* Remove the need to specify `unsafe-eval` when [Chart Templates]({%slug components/chart/label-template-format%}) are used.
+* Remove the need to specify `unsafe-inline`.
+* Fix the `font-src` rule by detaching font icons.
```
Comment (ntacheva) [inline troubleshooting/csp.md:46]:
```suggestion
* Remove the need to specify `font-src` rule by detaching font icons.
```

**Pull request merged**